packages/manager/modules/pci/src/projects/project/storages/containers/add/add.controller.js

```diff
@@ -3,7 +3,12 @@ import map from 'lodash/map';
 
 import Container from '../container.class';
 
-import { OBJECT_CONTAINER_TYPES } from '../containers.constants';
+import {
+  OBJECT_CONTAINER_OFFERS,
+  OBJECT_CONTAINER_OFFERS_LABELS,
+  OBJECT_CONTAINER_TYPE_OFFERS,
+  OBJECT_CONTAINER_TYPES,
```

Comment:
Can't the offers be returned by API ?

Reply:
Unfortunately, the APIs do not give us this info